pyne/partisn.py

```diff
@@ -101,12 +112,16 @@ def write_partisn_input(mesh, hdf5, ngroup, pn, **kwargs):
         true, a linearly spaced grid of starting points is used, with dimension 
         sqrt(num_rays) x sqrt(num_rays). In this case, "num_rays" must be a 
         perfect square.
-    
-    Returns
-    -------
-    PARTISN Input file named by 'input_file' above or the default name.
-        Note: read comments generated in file. Not all variables will be 
-        assigned that are necessary.
+    dg : record array, optional, default = None
+        The output of pyne.dagmc.discretize_geom(). Use this input option if 
+        discretize_geom() has already been run, to avoid duplicating this
+        expensive step.
+    find_per_coarse : int, optional, default = 1
```

Comment:
@makeclean Didn't we find when we were originally making input that the # of fine mesh intervals per coarse had to be at least two. @elliottbiondo does this work with just one fine mesh per coarse?

Also should be fine_per_coarse not find_per_coarse.